Workspaces/MiniWorld/Scripts/MiniWorld.cs

```diff
@@ -96,7 +96,7 @@ private void OnEnable()
 
 		private void OnDisable()
 		{
-			if ((referenceTransform.hideFlags & HideFlags.DontSave) != 0)
+			if ((referenceTransform.hideFlags & HideFlags.HideAndDontSave) != 0)
```

Comment:
That's a good question, actually. I changed it so that it would successfully, destroy, but the hideflags check has always been there, and I think you might be the one who originally wrote it.